app/api/v2/resource/phones.rb

```diff
@@ -44,6 +44,11 @@ def validate_phone!(phone_number)
                    type: String,
                    allow_blank: false,
                    desc: 'Phone number with country code'
+          requires :channel,
+                   type: String,
+                   allow_blank: false,
+                   values: { value: %w(sms call), message: 'resource.phone.invalid_channel'},
```

Comment:
better we define a range of values in the model and use it everywhere, so we will not miss it anywhere, if any changes.
like `Phone::TWILIO_CHANNELS`
(think about naming)